demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,221 @@
+
+**Note:**
+- To modify the program for quantizaiton, some training options need to disabled. Please disable some options as follows:
+```
+build_strategy.fuse_all_reduce_ops = False
+build_strategy.sync_batch_norm = False
+```
+- In `train_image_classification.py`, `paddleslim.quant.convert` is used to change the order of `fake_quantize`/`fake_dequantize` ops in Program. In addition, `paddleslim.quant.convert ` will change the operator parameters values to the values within the range of quantized `int8_t`, but the data type is still `float32`. This is the fp32 qat model we need, the default saving location is `./quantization_models/`.
+
+## 3. Convert fp32 qat model to INT8 model
+The model saved after training in the previous step is the float32 qat model. We have to remove the `fake_quantize`/`fake_dequantize` ops, and fully convert it into INT8 model. Go to the Paddle directory and run
+
+```
+python Paddle/python/paddle/fluid/contrib/slim/tests/save_qat_model.py --qat_model_path=/PATH/TO/FLOAT32/QAT/MODEL --int8_model_save_path=/PATH/TO/SAVE/INT8/MODEL --fp32_model_save_path=/PATH/TO/SAVE/FLOAT32/MODEL --ops_to_quantize="conv2d,pool2d"
+```
+**Available options in the above command：**
+- **qat_model_path:** The path where we save qat float32 model after training.
+- **fp32_model_save_path:** Optional. If set, meaning the path to the saved optimized fp32 model
+- **int8_model_save_path:** Optional. If set, meaning the path to the saved optimized and quantized INT8 model
+- **ops_to_quantize:**  A comma-separated list of operator types to quantize. If the option is not used, an attempt to quantize all quantizable operators will be made, and in that case only quantizable operators which have quantization scales provided in the QAT model will be quantized. When deciding which operators to put on the list, the following have to be considered:
+  - Only operators which support quantization will be taken into account.
+  - All the quantizable operators from the list, which are present in the model, must have quantization scales provided in the model. Otherwise, quantization of the operator will be skipped with a message saying which variable is missing a quantization scale.
+  - Sometimes it may be suboptimal to quantize all quantizable operators in the model (cf. Notes in the Gathering scales section above). To find the optimal configuration for this option, user can run benchmark a few times with different lists of quantized operators present in the model and compare the results. For Image Classification models mentioned above the list usually comprises of conv2d and pool2d operators.
+
+## 4. Inference test
+
+### 4.1 Data preprocessing
+To run the inference test, the data needs to be converted to binary first. Run the following script without any pramaters allows you to transform the complete ILSVRC2012_val_data set to bianary file. Use `local` parameter to transform your own data. Go to Paddle directory and run:
```

Comment:
... Run the following script without any parameters to transform the whole ILSVRC2012_val_data dataset into a binary file. ...